src/service.js

```diff
@@ -155,10 +193,18 @@ module.exports = function(mixinOptions) {
 					params: staticParams = {},
 					rootParams = {},
 					fileUploadArg = null,
+					fieldName = "",
+					typeName = "",
 				} = def;
 				const rootKeys = Object.keys(rootParams);
 
 				return async (root, args, context) => {
+					// Record the span if possible
+					let operationSpan;
+					if (context.ctx)
+						operationSpan = context.ctx.startSpan(`GQL ${typeName} ${fieldName}`);
```

Comment:
@icebob I am unfamiliar with how best to approach setting up spans for tracing, so I'm not sure if this is an appropriate pattern.  Can you weigh in?

Reply:
The method is correct but it would be good to add the `typeName` and `fieldName` into tags, as well.